### PR TITLE
eastwood less ugly

### PR DESCRIPTION
**Pull request by cornercube**
## About The Pull Request

exactly what is says on the tin

ground floor changes:
Jyn-Erso
flooring on the clinic changed for each room, so it isnt one big bathroom

Jyn-Erso
same with the bar, kitchen has a tiles because thats easier for people to clean and to create contrast with the bar area
each booth has carpet underneath to mark out its zone and tables replaced with booth tables because its a booth
a decorative set of X-1  prototype power armor hangs from a wall, anchored to prevent theft and to raise the ire of the BoS

Jyn-Erso
ghetto added to the dead end
has some free jet and hobo weapons

second floor changes
Jyn-Erso
the citizens hotel have customed rooms, each one has different contents 
nice and cozy

Jyn-Erso

clinic second floor has a bit more stuff in it like clothing vendos in addition to flooring changes
also gets a small psychiatrists room for one on one counseling
also gets a small office for the director with spare headsets and a recharger pistol

Jyn-Erso
bar second floor has themed rooms now
hallway floors is also different from the first
has a toilet now, for immulsions

not eastwood but worth noting:
Jyn-Erso
a way to get the enviro-suit through normal gameplay and a funny POI
shouldnt be an issue, since envirosuit has 35 resist and only 3 threshold

other changes:
2 compliance regulators spawn in the clinic security checkpoint
2 more spawn in the police armory
merchant has a recharger rifle to sell
a second vault-themed room in the sewers

big update: now has some overlapping changes with #324 
so that the role will not spawn in the middle of the desert
## Pre-Merge Checklist
- [ ] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by Sinbest** [comment]
This doesn't fix anything. The entire area needs to be ripped up and remapped from the beginning.

(or you could just bring back oasis but rebrand it as "eastwood" *cough *cough)

**Comment by cornercube** [comment]
> This doesn't fix anything. The entire area needs to be ripped up and remapped from the beginning.
that is not the point of this PR
its just a tweak to make things look a little better

any feedback should be made in the context of a tweak not a sweeping change

**Comment by Al-1ce** [comment]
Really not a fan of the kitchen having tiles.

**Comment by cornercube** [comment]
> Really not a fan of the kitchen having tiles. I think everything else looks great though.

any suggestions for the kitchen floor?

File: code/modules/jobs/job_types/eastwood.dm [hunk 1]
```diff
@@ -341,6 +343,76 @@ here's a tip, go search DEFINES/access.dm
...
+/*--------------------------------------------------------------*/
+
+/datum/job/eastwood/f13towncmo
+	title = "Director"
+	flag = F13CHIEFMED
+	display_order = JOB_DISPLAY_ORDER_F13CHIEFMED
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "The Mayor and Overseer"
+	description = "From the board of the VTCC, you were chosen to be the chief deligator for the medical and science wings of this town's clinic. You are the mediator of workplace drama, the human resource enforcer, and the cornerstone of inspiration for your underlings. As their supervisor, it is your job to make sure that this clinic functions at peak performance."
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/eastwood.dm:357]: delegator*

**Comment by Al-1ce** [comment]
> any suggestions for the kitchen floor?

Just wood! Really. It's fine as it is. Anything else and it'd feel out of place.

**Comment by cornercube** [comment]
> Just wood! Really. It's fine as it is. Anything else and it'd feel out of place.
ncr jpg

wood for kitchens isnt really ideal in real life compared to tile, since it will scratch and stain more easily
though the bigger issue is that we dont have enough wood floor types, hard to make it look different from the rest of the bar

**Comment by Al-1ce** [comment]
> wood for kitchens isnt really ideal in real life compared to tile, since it will scratch and stain more easily though the bigger issue is that we dont have enough wood floor types, hard to make it look different from the rest of the bar


Well, we live in a wasteland and are rarely ideal. Wherever the hell you would fine pristine tiles aside - I think the regular wood on both sides still looks much better after having seen this in game. Wouldn't be caught dead in the pale tile bar.

**Pull request merged**